src/stories/HeatmapVis.stories.tsx

```diff
@@ -19,7 +19,23 @@ const logSafeDomain = getDomain(dataValues, ScaleType.Log);
 const alphaArray = ndarray(dataValues.map((x) => Math.abs(x)));
 const alphaDomain = getDomain(alphaArray);
 
-const Template: Story<HeatmapVisProps> = (args) => <HeatmapVis {...args} />;
+const Template: Story<
+  Omit<HeatmapVisProps, 'scaleType'> & {
+    scaleType: ScaleType;
+    gammaExponent: number;
+  }
+> = (args) => {
+  const { scaleType, gammaExponent, ...otherArgs } = args;
+  const props = {
+    scaleType:
+      scaleType === ScaleType.Gamma
+        ? ([ScaleType.Gamma, gammaExponent] as [ScaleType.Gamma, number])
+        : scaleType,
+    ...otherArgs,
+  };
+
+  return <HeatmapVis {...props} />;
+};
```

Comment:
Some tweaking to be able to test the gamma scale in the Storybook

Reply:
I think it may be simpler to keep this template unchanged and just override the `scaleType` prop to only allow `ScaleType` values. Then, you can change the gamma scale story so it doesn't use the default template and so the gamma scale type is hardcoded. You can then add the `gammaExponent` control in that `stories` file and use it in the gamme scale story.

Reply:
Not sure I understood fully. Have a look at the latest changes and see if that is what you wanted

Reply:
Yep, looks good 😉  - you no longer have to export the `Template`, I think. Also, you may have to add a custom control for the `gammeExponent` to the gamma scale story.